server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/pipeline_configs/task_basic_views_widget.js.msx

```
@@ -309,8 +309,6 @@ define([
                     _.each(response.data, $scope.addError, $scope);
                   } else if (response.status === 520) {
                     $scope.pluggableTaskGenericError = response.data.error;
-                  } else {
-                    console.log('Something went wrong, we do not know what!');
```

Should we throw an alert here, instead of ignoring the error?